content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,117 @@
+
+- [NAT traversal](../nat/overview): WebRTC includes mechanisms (like
+  [ICE](https://datatracker.ietf.org/doc/rfc5245/))to connect to nodes that run behind
+  NATs and firewalls. In non-decentralized WebRTC, this can be facilitated by a
+  [TURN server.](https://webrtc.org/getting-started/turn-server),
+  but other signaling channels, such as WebSocket running on a central server, can also be used.
+  It is also possible to use a custom signaling protocol or a different signaling service.
+  Overall, this allows for faster and more efficient communication.
+
+- Security: WebRTC connections are encrypted using
+  [DTLS](https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security).
+
+Browsers expose an API to establish WebRTC connections. The
+[`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/RTCPeerConnection)
+API allows two applications on different endpoints to communicate. WebRTC enables
+a node to send and receive data through the `DataChannel`.
+
+### Browser-to-Server
+
+libp2p WebRTC enables browsers to connect to public server nodes (and eventually,
+browsers to connect to other browsers) without those endpoints providing a TLS
+certificate within the browser's trustchain.
+
+{{< alert icon="" context="info" text="When connecting to a WebSocket server, browsers require the server to present a TLS certificate signed by a trusted CA (certificate authority). Few nodes have such a certificate, which is the reason that WebSocket never saw widespread adoption in the libp2p network. WebRTC and WebTransport support encrypted communication without requiring a signed certificate from a trusted CA in the same way that WebSockets do." />}}
+
+In libp2p:
+
+- WebRTC multiaddresses are composed of a standard UDP multiaddr,
+  followed by `webrtc` and the `multihash` of the certificate that
+  the node uses, as such:
+  `/ip4/1.2.3.4/udp/1234/webrtc/certhash/<hash>/p2p/<peer-id>`;
+- WebRTC offers security via DTLS. Peers still need to authenticate remote peers
+  by their libp2p identity.
+
+A browser can connect to a server node without needing a trusted TLS
+certificate. View [this scenario](https://github.com/libp2p/specs/blob/master/webrtc/README.md#browser-to-public-server) on browser-to-server connection establishment
+in the technical specification.
+
+In general, a server will act as an [ICE Lite](https://www.rfc-editor.org/rfc/rfc5245)
+agent and binds to a UDP port waiting for incoming STUN and SCTP packets and multiplexes
+based on source IP and source port.
+
+When a browser wants to establish a WebRTC connection to a server, it instantiates
+an `RTCPeerConnection`. The browser constructs the server's SDP answer locally
+based on the browser's multiaddr. The browser creates a local offer via
+`RTCPeerConnection.createOffer()`. A sets the same username and password on the
+local offer as in the remote SDP answer.
```

Comment:
That's also very specific. Maybe we can say: 
Contrary to a standard WebRTC handshake, browser and server don't exchange the SDP. Instead, they apply a technique called "SDP munging" (link), which basically means that the browser only pretend that an SDP was exchanged, but in reality it constructed it locally using the information contained in the server's multiaddress.